src/components/editor/spec.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import test from 'tape';
+import { shallow, mount } from 'enzyme';
+import spy from 'utils/spy';
+
+import {
+  EditorState,
+  ContentState,
+  ContentBlock,
+  CharacterMetadata,
+  convertToRaw,
+  convertFromRaw,
+  genKey,
+} from 'draft-js';
+import { is, fromJS, List, Repeat } from 'immutable';
+
+import EditorFactory from './';
+import UpstreamEditor from 'draft-js-plugins-editor';
+
+const genContent = (text) => {
+  const contentState = ContentState.createFromBlockArray([
+    new ContentBlock({
+      key: 'abc',
+      type: 'unstyled',
+      text,
+      characterList: List(Repeat(CharacterMetadata.EMPTY, text.length))
+    }),
+  ]);
+  return convertToRaw(contentState);
+};
+
+const stateFromRaw = (raw) => {
+  return EditorState.createWithContent(convertFromRaw(raw));
+}
+
+const getState = (el) => {
+  return el.find('span').text();
+};
+
+test('Editor status', t => {
+  const Editor = EditorFactory(React);
+  t.plan(3);
+
+  const content = genContent('Write here');
+  const instance = shallow(<Editor content={content} />);
+
+  t.equals(getState(instance), 'Reading');
```

Comment:
Every assertion should include a human-readable description of that's being tested (the final argument to any tape assertion function).